pyo3-ffi/src/lib.rs

```diff
@@ -410,6 +410,8 @@ pub use self::enumobject::*;
 pub use self::fileobject::*;
 pub use self::fileutils::*;
 pub use self::floatobject::*;
+#[cfg(Py_3_9)]
```

Comment:
Once you've applied my other comments you should be able to delete this. We want to keep the config flags next to the FFI definitions. IMO it doesn't matter much if this is an empty module on Python 3.8.

Reply:
Done